Update use case pages to use shortcodes (#2588)
viamrobotics/docs: docs/use-cases/manage-fleet.md

You can monitor and teleoperate all of the robots from one online dashboard, and grant permission to other users to do the same.
You can grant users different levels of access to individual machines or to groups of machines.

{{< table >}}
{{< tablestep >}}
{{<imgproc src="/use-cases/signup.png" class="fill alignright" resize="600x" style="max-width: 350px" declaredimensions=true alt="Viam app signup screen">}}
**1. Create an account**

> Go to the [https://app.viam.com](Viam app) and sign up with Google, GitHub, Apple, or an email address.

Reviewer: ```suggestion
Go to the [Viam app](https://app.viam.com) and sign up with Google, GitHub, Apple, or an email address.
```

Assign each user a role (owner or operator) to manage permissions.

{{< /tablestep >}}
{{< tablestep >}}
{{<imgproc src="/fleet/app-usage/create-machine.png" class="fill alignleft" resize="600x" style="max-width: 450px" declaredimensions=true alt="Create a new machine in the Viam app">}}
**4. Connect machines to the cloud**

Users with access to a location can create and collaborate on the [machines](/fleet/machines/) within it.
When you [install `viam-server`](/get-started/installation/) on each machine, unique keys are generated to securely connect it to the Viam app.
Use the config builder interface to configure {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="services" >}} for new or existing smart machines.
You can use {{< glossary_tooltip term_id="fragment" text="fragments" >}} to streamline the process of configuring multiple similar machines.

{{< /tablestep >}}
{{< tablestep >}}
{{<imgproc src="/use-cases/last-online.png" class="fill alignright" resize="600x" style="max-width: 100px" declaredimensions=true alt="Machine last online status indicator in the Viam app.">}}
**5. Monitor your fleet**

> View the status and logs from each machine in the [https://app.viam.com](Viam app), gather machine sensor [data](/data/), and operate machines remotely from the [**Control** tab](/fleet/machines/#control).

Reviewer: Nit: what does "gather machine sensor data" mean over "view the status and logs in the Viam app" ?

Maybe something like:

```suggestion
In the [Viam app](https://app.viam.com), you can:
- Monitor the status of each machine from its [**Logs** tab](/fleet/machines/#logs).
- View any data captured by your fleet from the [**Data** tab](/data/).
- Operate machines remotely from the [**Control** tab](/fleet/machines/#control).
```

{{< /tablestep >}}
{{< tablestep >}}
{{<imgproc src="/fleet/app-usage/create-machine.png" class="fill alignleft" resize="600x" style="max-width: 450px" declaredimensions=true alt="Create a new machine in the Viam app">}}
**4. Connect machines to the cloud**

Users with access to a location can create and collaborate on the [machines](/fleet/machines/) within it.
When you [install `viam-server`](/get-started/installation/) on each machine, unique keys are generated to securely connect it to the Viam app.
Use the config builder interface to configure {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="services" >}} for new or existing smart machines.
You can use {{< glossary_tooltip term_id="fragment" text="fragments" >}} to streamline the process of configuring multiple similar machines.

{{< /tablestep >}}
{{< tablestep >}}
{{<imgproc src="/use-cases/last-online.png" class="fill alignright" resize="600x" style="max-width: 100px" declaredimensions=true alt="Machine last online status indicator in the Viam app.">}}
**5. Monitor your fleet**

Using the [Viam app](https://app.viam.com), you can:

> - Monitor the status of each machine from its [**Logs** tab](/fleet/machines/#logs).
> - View any data captured by your fleet from the [**Data** tab](/data/).
> - Operate machines remotely from the [**Control** tab](/fleet/machines/#control).
> Use [modules](/registry/) to deploy code to your fleet and manage versioning.

Author: ```suggestion

- Monitor the status of each machine from its [**Logs** tab](/fleet/machines/#logs).
- View any data captured by your fleet from the [**Data** tab](/data/).
- Operate machines remotely from the [**Control** tab](/fleet/machines/#control).

Use [modules](/registry/) to deploy code to your fleet and manage versioning.
```
